data/migrations/20201109234734_stores-orders-products-tags.js

```diff
@@ -17,6 +17,29 @@ exports.up = function (knex) {
       tb.string('operating_hours', 255);
       tb.date('created_at');
     })
+    .createTable('orders', (tb) => {
+      tb.increments();
+      tb.integer('store_id')
+        .unsigned()
+        .notNullable()
+        .references('id')
+        .inTable('stores')
+        .onDelete('CASCADE')
+        .onUpdate('CASCADE');
+      tb.string('buyer_id')
+        .notNullable()
+        .references('id')
+        .inTable('profiles')
+        .onDelete('CASCADE')
+        .onUpdate('CASCADE');
+      tb.string('ship_to');
+      tb.string('delivery_method').notNullable();
+      tb.string('delivery_service');
+      tb.integer('shipping_cost');
+      tb.string('tracking_number');
+      tb.string('status');
+      tb.date('created_at');
+    })
     .createTable('tag', (tb) => {
       tb.increments();
       tb.string('tag_name', 255);
```

Comment:
lets change this property to just name - we want consistency - on the products table we say "name" only but here "tag_name"